doc/integrator/caching.rst

```diff
@@ -8,12 +8,10 @@ Headers
 
 By default most of the elements will be cached for 10 days by the browser.
 
-To change this value for all the application change in the
-``vars.yaml`` file the ``vars/default_max_age`` value. ``0`` will mean ``no-cache``.
-The specified value is in seconds.
+To change this value for all the application change in the ``vars.yaml`` file the ``vars/default_max_age``
```

Comment:
``vars/headers/<service_name>/cache_control_max_age`` value

Reply:
Oups I should change this but booth are used but in defferent way...

Reply:
Just updated :-)